modules/gltf/extensions/physics/gltf_physics_body.cpp

```diff
@@ -116,15 +144,14 @@ Ref<GLTFPhysicsBody> GLTFPhysicsBody::from_node(const CollisionObject3D *p_body_
 	if (cast_to<CharacterBody3D>(p_body_node)) {
 		physics_body->body_type = "character";
 	} else if (cast_to<AnimatableBody3D>(p_body_node)) {
-		physics_body->body_type = "kinematic";
+		physics_body->body_type = "animatable";
```

Comment:
I do not understand where the string `animatable` comes from.

I think we should stick for now with what is in the schema and add this as extras or a separate GODOT_ extension if needed
https://github.com/omigroup/gltf-extensions/blob/main/extensions/2.0/OMI_physics_body/schema/node.OMI_physics_body.motion.schema.json#L12..L16

Reply:
As I have explaines before, the in-memory type list is closer to Godot's types for flexibility, and is expanded on import / flattened on export. We only ever deal with the types in the OMI schema at the level of glTF files.

This allows reading "dynamic" from OMI data, then setting it in memory as "rigid", then other extensions can change this to "vehicle". Or in this case, reading "kinematic", in memory "animatable", and other extensions can change this to "character".

This mechanism exists precisely to allow a separate extension to exist in the future, with code to support new types not needing to fight with the rest of the physics code.